### ISSUE TITLE
[Bug]: Tooltip missing for password input field (Accessibility bug)

### ISSUE THREAD

**Issue by xiangj1** [microsoft/fluentui#35150]
### Component

Accordion

### Package version

9.0.0

### React version

18

### Environment

```shell
Generic error that also repro in demo page on fluent 
https://react.fluentui.dev/?path=/docs/components-input--docs#type
```

### Current Behavior

No tooltip showing

### Expected Behavior

tooltip show

### Reproduction

https://stackblitz.com/edit/or9tsbte?file=package.json

### Steps to reproduce

https://react.fluentui.dev/?path=/docs/components-input--docs#type

Type password in the password field and hover on the eye icon

### Are you reporting an Accessibility issue?

None

### Suggested severity

Medium - Has workaround

### Are you willing to submit a PR to fix?

no

### Validations

- [x] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [x] The provided reproduction is a minimal reproducible example of the bug.

**Comment by tudorpopams** [comment]
@ValentinaKozlova , @smhigley is this something intended?

**Comment by ValentinaKozlova** [comment]
I think there should be a tooltip and an aria-label for icon. @smhigley , what do you think?
Also, what expected behavior for screen readers when a user clicks on "eye" icon? Should it read the password?

**Comment by dmytrokirpa** [comment]
I don't think it is a bug, as we don't provide a password toggle functionality out-of-the-box, and that's the browser-specific behavior (most likely Edge). 

But you could easily achieve this by composing Input + Tooltip + Toggle as demonstrated here https://stackblitz.com/edit/or9tsbte-npzp9ig3?file=src%2Fexample.tsx

Demo:

https://github.com/user-attachments/assets/93fb4941-967a-49b2-98c0-a6fef2d51f56

Hope that helps, and please me know if you have other questions!
